### ISSUE TITLE
RGH: input[type=text] fields lack border

### ISSUE THREAD

**Issue by fregante** [StylishThemes/GitHub-Dark#1067]
* **Browser**: Firefox 73
* **Operating System**: macOS 10.15
* **Link to page with the issue**:
	- https://github.com/StylishThemes/GitHub-Dark/issues/new (title field)
	- https://github.com/StylishThemes/GitHub-Dark/issues (search field)
	- when merging a PR (merge commit title field)

## Initial

<img width="747" alt="border" src="https://user-images.githubusercontent.com/1402241/74862816-bde7d900-5387-11ea-841e-b01e4c88ed42.png">

## Current

<img width="749" alt="no border" src="https://user-images.githubusercontent.com/1402241/74862820-c04a3300-5387-11ea-9e45-84a27eb68e60.png">

**Comment by fregante** [comment]
Also the border around "Add more commits..." makes it look too important. It's supposed to be a small note (normally it's smaller and gray) but now it looks like a section on its own

**Comment by silverwind** [comment]
Looks to be RGH related.

**Comment by fregante** [comment]
It's related, here's GitHub-Dark's CSS

```css
.uploaded-files.is-populated, html.refined-github input:not([type="checkbox"]):not(:placeholder-shown):valid, html.refined-github input:not([type="checkbox"]):invalid, body.zh_logged_in .zhc-color-square, body.zh_logged_in .zhc-workspace-switcher-item, body.zh_logged_in .zhc-roadmap-selection-item {
    border-color: 
    #181818;
}
```

**Comment by silverwind** [comment]
@fregante was `input:not([type="checkbox"]):invalid` recently removed in RGH?

**Comment by silverwind** [comment]
Oh, I see it's coming from `source/options.css` which we should not apply on content.

**Comment by silverwind** [comment]
Limited CSS extraction to only what's in [content_scripts](https://github.com/sindresorhus/refined-github/blob/01bfaef2cf32b3234221cebe80164bb04f573109/source/manifest.json#L42) for RGH.

The reason it previously pulled all CSS was that some other extensions did not list their content-applied CSS files in `content_scripts`. I hope you will never do that in RGH 😉

**Comment by silverwind** [comment]
Both fixed and released in v1.22.223.
